src/app/components/form-manager/start-new-form/patient-summary-table/patient-summary-table.component.html

```diff
@@ -30,10 +30,10 @@
       [class.selected-patient-row]="(selectedPatient$ | async) === row"
       (click)="setSelectedPatient(row)">
   </tr>
-
+<!--  TODO: Render only after a search has been executed-->
   <tr class="mat-row" *matNoDataRow>
-    <td class="mat-cell no-data" [attr.colspan]="displayedColumns.length">
-      No SmartChart Forms Found.
+    <td class="mat-cell" [attr.colspan]="displayedColumns.length">
+      <span class="no-data-table-row">No records found</span>
```

Comment:
Mentioned in slack: does not patch the patient-summary-table scss class name. (Which is .no-data-row, not .no-data-table-row)